cura/CuraApplication.py

```diff
@@ -186,7 +187,7 @@ def __init__(self, *args, **kwargs):
 
         self._cura_formula_functions = None  # type: Optional[CuraFormulaFunctions]
 
-        self._machine_action_manager = None  # type: Optional[MachineActionManager.MachineActionManager]
+        self._machine_action_manager = None  # type: Optional[MachineActionManager]
```

Comment:
Use modern python typing
```suggestion
        self._machine_action_manager: Optional[MachineActionManager] = None
```
We didn't do this throughout the codebase as it wasn't always available, but for newly added code we should always use modern typing